x-pack/plugins/remote_clusters/public/app/services/notification.ts

```diff
@@ -4,4 +4,10 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
-export const CRUD_APP_BASE_PATH = '/management/elasticsearch/remote_clusters';
+export let toasts: any;
+export let fatalError: any;
+
+export function init(_toasts: any, _fatalError: any): void {
+  toasts = _toasts;
```

Comment:
my preference is to keep this name as `toastNotifications` so that this app's usages of toasts show up when grepping for all `toastNotifications` references in the codebase